app/pb_kits/playbook/pb_nav/docs/example.yml

```diff
@@ -3,5 +3,6 @@ examples:
   - default_nav: Default
   - block_nav: Block
   - block_no_title: Without Title
+  - horizontal_nav: Horizontal Nav
```

Comment:
It would be great to have this example right after the default. Then go into other settings